products/statement-generator/src/styles/utilityStyles.ts

```diff
@@ -48,6 +48,7 @@ const useUtilityStyles = makeStyles<Theme>(
         flexDirection: 'row',
         width: '100%',
       },
+
```

Comment:
@davidwiese seems silly, but lets remove this added line so you aren't making changes to this file at all.